feat: add support for Kafka secure connection with certificates X509 (#70)
asyncapi/nodejs-template: README.md

* [Kafka](https://en.wikipedia.org/wiki/Apache_Kafka)
* [WebSocket](https://en.wikipedia.org/wiki/WebSocket)

## How to use the template

This template must be used with the AsyncAPI Generator. You can find all available options [here](https://github.com/asyncapi/generator/).

> In case you use X509 security and need to provide certificates, place them in the root of generated server with the following names: `ca.pem`, `service.cert`, `service.key`. Since you can have multiple different security schemes, to use the one of X509 type, you need to pass the name fo the scheme like this: `-p securityScheme=SCHEME_NAME`.

Reviewer: ```suggestion
In case you use X509 security and need to provide certificates, place them in the root of generated server with the following names: `ca.pem`, `service.cert`, `service.key`. Since you can have multiple different security schemes, to use the one of X509 type, you need to pass the name of the scheme like this: `-p securityScheme=SCHEME_NAME`.
```

> In case you use X509 security and need to provide certificates, place them in the root of generated server with the following names: `ca.pem`, `service.cert`, `service.key`. Since you can have multiple different security schemes, to use the one of X509 type, you need to pass the name of the scheme like this: `-p securityScheme=SCHEME_NAME`.

Reviewer 2: As a suggestion: Could we provide the absolute path instead? I think it would be easier for the users and simplifies a lot the file handling. Most of this paragraph will be then removed.

Reviewer 2: I just realized @fmvilas wrote some similar suggestion at https://github.com/asyncapi/nodejs-template/pull/70#pullrequestreview-683052861

Author: option to provide custom location for the files is valid, but I don't get how will it affect this paragraph, other then extend it more 😄

Reviewer 2: ```suggestion
In case you use X509 security and need to provide certificates, specify their location at BLA BLA. Since you can have multiple different security schemes, to use the one of X509 type, you need to pass the name of the scheme like this: `-p securityScheme=SCHEME_NAME`.
```

Author: for me best DX is defaults and later config. Description updated
